### DISCUSSION TITLE
Sobre o projeto MAPOS.

### DISCUSSION THREAD

**Discussion by Edivandro** [RamonSilva20/mapos#1905] [General]
Olá  ...
A seguinte idagação me ocorre, o que está sendo implementado no momento ? 
Como as futuras funcionalidades e tarefas são organizadas ?
Pois se uma pessoa em particular começa a criar alguma  'funcionalidade' e não há uma sincronia com os outros dev, acaba que que muitos tentam criar a mesma coisa é muitas vezes acaba não terminando tbm.
Alguém fornece a MODELAGEM desse Projeto ?
Quais  as CLASSES e estruturaBD e como elas estão relacionadas ? 

Tenho algum conhecimento em PHP, não tenho experiência em projetos, nenhuma prática profissional. Mas acredito poder contribuir, mas me sinto perdido a começar e um pouco confuso em como esse projeto funciona ?

**Comment by SuperPorte** [comment]
Eu também queria ajudar, mas meu último contato com programação foi turbo pascal e html.... lá em 98......

**Comment by edmarcos1972** [comment]
Também gostaria de me envolver no projeto. Até contriui por uns meses no catarse mas ainda não consegui usar o sistema em minha empresa ...
